recipes/dd-agent.rb

```diff
@@ -53,6 +53,11 @@
 #
 raise "Add a ['datadog']['api_key'] attribute to configure this node's Datadog Agent." if node['datadog'] && node['datadog']['api_key'].nil?
 
+api_keys = node['datadog']['api_key']
+dd_urls = node['datadog']['dd_url']
```

Comment:
actually it's probably not intentional since you're using `node['datadog']['url']` in `recipes/dd-handler.rb`, so let's fix it :)